packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -411,22 +428,18 @@ export const metricsEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
     tap(([action, state]) => {
       const connectionLabels = (payload: WSConfigPayload) => ({
         key: payload.config.connectionInfo.key,
-        url: censor(payload.wsHandler.connection.url),
```

Comment:
Unsure what to do with these.
I took them out unless someone can think of a better way.

Reply:
What's wrong with having them in?